src/status_im/browser/core.cljs

```diff
@@ -135,28 +135,37 @@
    {:keys [url] :as bookmark}]
   {:db            (assoc-in db [:bookmarks/bookmarks url] bookmark)
    ::json-rpc/call [{:method "browsers_storeBookmark"
+                     :params [bookmark]
+                     :on-success #(re-frame/dispatch [:browser/sync-bookmark %1])}]})
```

Comment:
why don't we sync it in status-go when storing ?

Reply:
it's a little hard to access `messenger` in services/browsers/service.go in status-go to implement sync when storing, it's easier to expose `SyncBookmark` in ext api for status-react to invoke. Should i make it happen to access `messenger` in services/browsers/service.go? @flexsurfer

Reply:
its up to you, cc @cammellos

Reply:
Ah, no, better that it's all handle in status-go, there are patterns for that, it's a bit cumbersome, but we should not leave the responsibility to clients